test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -87,7 +87,8 @@ private Random getRandomNumberGenerator() {
         return new Random(SEED);
     }
 
-    final int basePort = 9000;
+    private final int basePort = 9000;
+    private final int retries = 5;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651444)